test/e_chronicler/models/journal_entry_test.exs

```diff
@@ -4,13 +4,16 @@ defmodule EChronicler.Models.JournalEntryTest  do
 
   alias EChronicler.Models.JournalEntry
 
+  @valid_journal_entry %{title: "Awesome Blog Post", author: "Grace", entry: "Time is an illustion, teatime doubly so."}
+  @invalid_journal_entry %{title: "", author: "", entry: ""}
```

Comment:
A bit nit picky on my part -- but are these each a `valid/invalid_journal_entry or a`valid/invalid_journal_attrs`?  I'd think the latter...like, I'd expect a struct (`%JournalEntry{...stuff...}`) if these were journal entries.  Same with the other tests.

Reply:
That is a good point! I'll change that.